File: bit-python01/src08/ex05/calculator2.py
# 클래스 상속

from ex05.calculator1 import Calculator1

class Calculator2(Calculator1):
    '''클래스에 대한 간단한 설명'''
    
    def multiple(self, value):
        '''곱하기를 수행하는 메서드'''
        self.result *= value
    
    
    
    

File: bit-java01/src/main/java/step11/ex3/Calculator2.java
package step11.ex3;

// 상속은 기존 코드를 손대지 않고 기능을 확장할 수 있다.
public class Calculator2 extends Calculator {
  void multiple(float value) {
    this.result *= value;
  }
  
  void divide(float value) {
    this.result /= value;
  }
}
